src/actions/editUserData.ts

```diff
@@ -52,13 +54,21 @@ export const editUserData = async (
       };
     }
   }
+
+  let uploadedAvatarUrl;
+
+  if (avatar) {
+    uploadedAvatarUrl = (await uploadImageToCloudinary(avatar)).url;
+  }
```

Comment:
i believe it can be simplified, also can we use `secure_url` or there is some reason to use `url`?
```suggestion
  const uploadedAvatarUrl = avatar
    ? (await uploadImageToCloudinary(avatar)).secure_url
    : null;
```

Reply:
i was sure that it returns https out of the box, thanks xD